css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,19 @@
+button {
+  padding: 20px;
+  margin-left: 30px;
+  font-size: medium;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+}
+
+th:nth-child(n) {
+  background-color: black;
+  color: white;
+}
```

Comment:
The `nth-child` pseudo-class isn't necessary if you selected every single one. Also, there's one one.